### ISSUE TITLE
Chokidar watch not working on non-C drive without polling

### ISSUE THREAD

**Issue by bjg2** [paulmillr/chokidar#998]
**Describe the bug**

Watch works for C drive but not for E drive, if polling is not added. Note that E is a regular drive and not a 

**Versions (please complete the following information):**
 - Chokidar version: 3.3.1
 - Node version: 12.16.1
 - OS version: Microsoft Windows Server 2019 Standard

**To Reproduce:**

I know this is not the best repro, but either it is the same everywhere or just on this machine for some reason. Watch on drive C works without added lines below, but on E it does not (neither event is fired).

![image](https://user-images.githubusercontent.com/6639273/77944321-0b2c6400-72bf-11ea-99e6-557701ce2c48.png)

**Expected behavior**
Expected was that watching a file on E drive works without polling as well.

**Additional context**
Machine in question is a machine with 3 SSDs on hetzner.

**Comment by paulmillr** [comment]
How does watchPaths look like?

**Comment by bjg2** [comment]
It is a string to a single file, something like `e:/tasks/5e4e898e199f795e9a1ce4d9/runs/0/output.json`

**Comment by paulmillr** [comment]
Note that you need to use `\` in paths on Windows.

**Comment by cdaven** [comment]
> Note that you need to use `\` in paths on Windows.

This seems wrong? The README says:

> Note: globs must not contain windows separators (\), because that's how they work by the standard — you'll need to replace them with forward slashes (/).

Also, using `\` doesn't work when I try it, but `/` does.

**Comment by PirateMx** [comment]
I wanted to add some updates for this. I ran into issues where chokidar would not watch a folder properly that was a network drived mapped to Z:

Everything worked fine locally with the mapped drive and on a remote computer when run from CMD with node.js. However, when I used NSSM (Non-sucking service manager) to run the node app in a window service I had issues. I added logging to determine that my watcher.getWatched() returning nothing even though the ready event fired. I had the drived mapped properly and eveything. It worked perfectinly in CMD from the same computer.

I ended up switching to network path and was able to get it working with \\\\PC_NAME\\Folder\\Folder for my path when running in NSSM for my windows service. Not sure if there is a configuration I was missing that would allow the service to see the mapped drive or not. I used an Administrator account and everything.
